Apps/WebClient/src/ClientApp/src/stores/report.ts

```diff
@@ -16,35 +16,35 @@ export const useReportStore = defineStore("reports", () => {
         if (dependent) {
             return {
                 phn: dependent.dependentInformation.PHN,
-                dateOfBirth: DateWrapper.format(
-                    dependent.dependentInformation.dateOfBirth ?? ""
-                ),
+                dateOfBirth: dependent.dependentInformation.dateOfBirth
```

Comment:
NIT:
I've seen this ternary structure in many places. What about considering `.format()` returning null if DateWrapper is invalid.

```TypeScript
DateWrapper.fromIsoDate(dependent.dependentInformation.dateOfBirth).format() ?? ""
```